frontend/src/components/phones/dashboard/PhoneDashboard.module.scss

```
@@ -404,20 +404,48 @@
 
       .phone-controls {
         .forwarding-toggle {
+          width: $content-sm;
+          max-width: 100%;
+          background-color: $color-grey-05;
           border-radius: 40px;
-          border: $border-radius-xs solid $color-light-gray-20;
-          padding: $spacing-sm;
+          border: 6px solid $color-white;
+          outline: $border-radius-xs solid $color-grey-10;
           display: flex;
           align-items: center;
           justify-content: center;
-          gap: $spacing-md;
+          position: relative;
+
+          // This is the background for the selected state; it's a separate
+          // element so we can animate it moving behind the different state
+          // labels ("Forwarding" and "Blocking"):
+          &::before {
+            content: "";
+            position: absolute;
+            background-color: $color-violet-70;
+            border-radius: 40px;
```

_Magic Number™_ Question/Suggestion: Should we use a [Protocol border radius token](https://protocol.mozilla.org/docs/fundamentals/design-tokens.html#border-radii) here and multiply it? I'm not sure if I feel like we should do this, but wanted to get your take. 
```suggestion
            border-radius: calc((#{$border-radius-md} * 2)
```

---

Yeah good question. I'm not generally a fan of doing math on tokens unless that math actually expresses what we want to do (i.e. did design explicitly choose a border radius that's twice the medium border radius, or did they just come up with an arbitrary number that looked good - I think in this case it's the latter). That said, this is absolutely a good callout that at least this should be in a variable, because this and the `::before`'s border-radius _should_ be the same. Fixed that in 2593615a.